external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitterEmitTest.java

```diff
@@ -81,15 +88,20 @@ public void setUp() {
         records.forEach(record -> consumer.addRecord(record));
     }
 
+    private KafkaTridentSpoutEmitter<String, String> createEmitter(Consumer kafkaConsumer, FirstPollOffsetStrategy firstPollOffsetStrategy) {
```

Comment:
Raw type. Please add generics to the Consumer.

Reply:
ok